styles/rentry.co/catppuccin.user.css

```
@@ -0,0 +1,558 @@
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/rentry.co
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/rentry.co/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Arentry.co
+@description Soothing pastel theme for Rentry.co
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('rentry.org'), domain('rentry.co') {
+  .dark-mode {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root:not(.dark-mode) {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    body {
+      background: @crust;
+      color: @text;
+    }
+    .cm-s-default,
+    .nav-link.active,
+    .entry-text,
+    .markdownx-preview {
+      background: @base !important;
+      color: @text !important;
+    }
+    .nav-link {
+      background: @mantle;
+      color: @overlay0 !important;
+    }
```

I would prefer if this was organized and commented/explained like this:
(I removed `.entry-text`, I don't know what it is or what it themes.)
```suggestion
    /* Editor and preview */
    .cm-s-default,
    .markdownx-preview {
      background: @base !important;
      color: @text !important;
    }

    /* Tab navigation */
    .nav-link {
      background: @mantle;
      color: @overlay0 !important;
      
      &.active {
        background: @base !important;
        color: @text !important;
      }
    }
```

---

`.entry-text` themes the background and text color of posts (non editor/preview pages) and official pages like [/what](https://rentry.co/what). it also requires `!important`
can you add it back so i can commit the suggestion

---

Done!